doc/index.rst

```diff
@@ -33,6 +33,11 @@ PennyLane-SF provides two Strawberry Fields devices for PennyLane:
     :description: Optimized simulator that supports only Gaussian operations and photon number resolving measurements.
     :link: devices/gaussian.html
 
+.. devicegalleryitem::
+    :name: 'strawberryfields.remote'
```

Comment:
If this is the final proposed name, make sure to update the device page to change `strawberryfields.ai`!

Reply:
Thanks! Should have changed it all